backend/app/services/audio/speech_to_text.py

```
@@ -0,0 +1,630 @@
+import os
+import logging
+import csv
+import base64
+import requests
+from io import StringIO
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import BinaryIO, Literal
+from pydantic import BaseModel
+from dotenv import load_dotenv
+from uuid import uuid4
+from typing import List, Literal, Dict
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)